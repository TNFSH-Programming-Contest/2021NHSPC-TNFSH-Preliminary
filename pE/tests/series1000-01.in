999 998
3 4 8
5 6 10
6 3 27
4 7 8
7 8 17
8 9 27
10 11 23
12 13 8
14 12 30
15 10 19
16 17 12
18 5 5
19 20 24
21 22 30
9 14 16
13 15 6
23 24 27
25 21 26
26 23 27
22 18 16
20 27 13
28 19 8
27 16 16
11 29 27
24 25 6
30 26 12
17 30 19
31 28 23
32 31 16
33 32 25
29 34 12
35 33 29
36 35 18
37 36 6
38 37 12
39 38 13
40 41 5
42 39 12
43 40 23
41 42 17
44 43 17
45 46 26
47 44 8
46 47 24
48 49 5
50 48 26
49 51 17
51 52 28
34 50 11
53 45 22
54 53 29
55 56 25
57 54 25
58 59 27
60 61 30
62 63 8
56 58 6
64 60 30
65 62 18
66 67 15
59 64 13
68 69 14
63 55 10
67 70 19
71 65 16
72 68 28
70 71 17
73 57 12
74 72 18
61 75 7
76 66 12
75 74 10
69 77 5
78 79 15
79 76 10
80 81 17
82 73 7
83 84 29
85 82 11
77 80 28
86 87 12
84 86 17
88 78 22
89 85 30
81 90 5
91 92 26
93 94 20
95 83 27
96 97 10
98 93 30
94 99 17
100 101 18
102 103 30
87 96 9
99 104 8
105 100 14
106 107 16
108 102 5
92 89 18
90 109 19
103 106 14
110 98 27
111 112 26
107 113 30
104 91 8
114 115 28
116 117 22
118 119 18
120 88 25
113 110 9
121 122 14
101 95 14
52 123 13
122 120 26
119 108 28
97 121 24
124 125 6
109 118 14
125 126 11
126 127 21
112 116 20
128 129 12
130 111 28
115 131 10
132 133 22
129 114 9
134 135 15
136 137 28
133 130 12
135 138 13
139 105 24
117 134 6
140 136 30
137 128 8
127 132 25
131 141 12
123 140 12
142 143 21
138 142 12
143 139 17
144 124 10
145 146 22
147 148 21
149 150 10
146 151 16
152 153 19
154 155 28
141 156 17
157 158 23
150 159 18
158 160 10
156 149 5
161 162 14
163 164 15
159 152 11
164 165 26
153 145 16
160 154 11
166 147 20
165 166 6
162 167 9
148 161 27
168 169 6
170 168 30
171 163 17
172 173 26
167 172 25
174 170 28
151 171 13
175 176 9
173 177 22
169 178 23
155 174 5
178 175 24
176 144 9
179 157 7
177 180 7
181 179 24
180 182 27
183 184 15
185 186 27
186 187 13
188 185 23
189 188 27
190 183 8
191 192 23
193 194 5
187 195 17
196 189 20
182 197 13
195 190 26
197 193 26
192 181 16
184 198 12
199 196 6
200 191 26
201 202 28
203 201 7
202 204 19
205 203 6
198 205 13
206 200 20
194 207 16
204 208 14
209 210 19
207 199 24
211 209 22
210 206 22
208 212 8
212 213 14
214 211 22
215 216 28
216 217 14
217 218 23
218 219 14
213 215 21
220 221 9
222 223 22
224 225 9
226 224 18
221 214 29
227 220 13
225 227 5
219 228 29
223 226 24
228 229 19
230 231 28
232 230 17
233 234 18
235 233 24
236 222 18
237 238 13
234 237 16
238 236 5
231 239 6
229 232 8
240 241 8
241 242 10
243 244 20
245 246 25
242 245 6
247 248 6
249 235 24
248 249 27
250 251 27
252 247 11
253 254 16
251 240 24
246 253 5
239 243 6
255 252 9
256 250 7
257 256 22
258 259 20
260 258 29
261 260 18
262 261 10
244 257 13
263 264 13
265 266 21
267 268 30
269 270 5
271 272 22
273 274 15
275 271 16
266 276 29
277 278 11
279 280 16
281 282 5
274 283 29
272 284 30
276 285 7
283 286 23
284 287 30
259 288 25
268 289 7
290 291 14
292 293 27
289 262 16
280 267 18
294 281 24
295 263 12
293 296 10
287 297 11
298 295 22
299 300 13
301 275 9
296 302 11
302 303 8
304 273 10
286 299 11
305 306 15
285 307 17
308 309 27
310 311 28
312 313 22
314 292 21
315 316 24
317 318 18
306 319 15
264 320 12
307 321 10
322 323 8
278 324 18
325 265 16
309 314 21
326 294 11
311 305 23
316 327 6
328 329 5
303 279 17
330 331 10
331 315 19
313 325 30
332 255 7
333 330 15
300 301 9
254 334 22
335 269 14
319 336 8
334 337 19
338 333 16
339 298 30
297 339 27
340 341 22
342 308 11
343 326 23
324 344 6
270 345 29
341 346 18
347 348 13
349 350 30
345 343 8
351 352 14
353 304 23
318 354 16
355 328 10
288 310 18
352 356 7
357 290 11
358 351 22
282 358 25
346 355 24
359 360 21
360 342 29
356 361 12
350 362 13
327 359 27
363 364 20
344 365 16
361 366 19
320 367 11
368 369 21
370 371 30
321 332 23
365 372 11
367 373 20
374 312 14
348 375 24
376 377 27
378 335 8
323 379 7
379 338 16
380 381 29
362 380 14
381 382 23
369 383 20
329 384 7
337 385 9
386 368 6
387 378 18
388 389 16
373 340 26
390 322 21
336 391 24
392 393 14
394 395 22
389 347 26
382 376 26
396 370 24
397 398 5
391 397 15
385 399 5
400 349 6
401 402 5
366 403 23
384 390 17
383 404 11
405 406 9
395 317 14
407 408 8
409 400 7
410 387 8
411 412 24
413 357 7
414 410 28
372 415 23
416 414 9
417 374 6
418 392 18
406 419 17
420 409 22
421 422 10
423 353 12
424 425 7
426 427 24
402 428 15
377 401 6
429 363 7
403 430 11
431 432 26
371 433 9
434 411 9
415 394 20
425 435 24
436 437 18
354 429 7
291 417 27
438 439 29
440 407 18
441 423 25
422 442 23
404 421 14
443 440 19
430 413 27
399 436 20
444 388 18
408 277 21
445 396 14
446 416 18
447 386 6
448 449 7
364 450 17
451 452 24
453 454 27
455 405 27
456 457 20
458 445 23
459 460 29
461 462 23
463 459 13
437 443 16
460 431 27
375 464 14
432 465 17
450 466 13
454 467 8
427 468 10
435 469 8
442 470 6
428 471 30
472 473 22
468 474 20
466 475 23
475 447 6
476 463 19
477 446 5
471 455 14
478 479 21
480 426 9
481 444 17
479 482 5
467 483 26
452 484 9
485 458 17
398 486 27
474 438 11
487 480 19
488 453 30
489 490 24
465 481 21
464 420 26
433 448 18
484 489 21
470 478 28
491 492 20
473 493 23
486 494 18
462 456 15
495 424 15
494 496 28
482 497 18
498 499 13
500 501 12
502 503 7
504 505 10
506 507 14
496 502 22
508 509 9
483 500 9
457 510 20
449 511 24
493 506 17
505 512 26
393 513 10
412 514 18
515 516 25
517 518 23
469 519 11
516 520 17
519 498 7
521 434 26
497 522 27
509 461 16
520 523 17
524 521 17
522 525 20
526 477 28
527 528 28
511 487 28
529 530 14
531 532 10
533 534 17
535 504 14
513 536 11
537 488 10
523 531 17
439 515 23
510 495 12
538 485 12
512 539 13
540 541 26
518 537 8
542 529 30
543 544 8
545 546 14
547 548 9
549 527 17
550 551 28
552 451 13
539 553 22
541 554 10
548 555 7
501 556 6
492 533 11
551 543 19
499 557 27
558 559 11
560 561 6
514 562 13
553 563 9
507 564 28
565 538 22
566 567 25
561 568 23
569 570 25
532 476 29
571 558 11
554 545 21
534 566 9
572 573 8
574 540 6
575 550 25
576 577 21
563 552 8
578 579 6
536 580 7
581 526 13
582 583 10
584 576 9
570 584 10
564 585 7
583 586 12
587 581 13
588 517 20
546 535 20
577 589 15
590 591 19
592 587 18
593 491 26
559 560 8
594 418 19
595 596 15
586 578 8
597 549 9
598 441 15
599 472 6
555 565 12
573 600 14
525 571 9
601 547 22
530 602 18
589 603 5
604 524 11
605 606 5
600 607 16
608 593 25
609 592 6
610 604 25
557 597 12
611 569 21
612 613 26
602 614 13
615 616 18
617 618 25
619 620 5
544 609 6
568 621 30
622 588 6
623 624 20
618 625 16
556 626 6
627 628 14
567 542 27
629 630 24
631 632 12
613 619 11
606 633 19
614 634 10
624 635 5
607 636 10
580 637 23
620 638 15
635 639 30
596 508 25
633 575 18
640 610 30
621 641 26
637 642 26
490 643 15
503 644 28
645 623 19
646 647 30
648 649 17
636 650 28
650 651 8
652 653 11
654 655 20
647 656 13
419 652 30
626 640 18
657 658 5
603 601 12
659 660 22
579 661 24
632 662 14
641 594 15
638 663 25
664 665 24
666 608 14
562 667 9
585 668 23
616 631 17
669 617 11
630 670 27
671 664 10
668 615 12
644 572 25
672 622 13
673 674 28
665 673 14
675 676 22
628 598 26
655 612 19
677 654 27
678 672 11
679 680 27
681 666 8
591 682 22
683 671 6
642 574 6
684 685 10
667 595 24
686 687 6
649 688 19
663 689 9
690 691 22
643 648 17
670 692 22
693 659 18
528 686 13
694 669 24
634 611 26
660 605 15
695 677 18
696 697 17
651 698 17
699 700 20
656 701 19
661 702 12
700 703 22
704 683 19
685 705 30
706 707 26
702 708 22
658 709 18
674 710 8
692 679 21
625 711 12
662 695 9
653 706 12
712 696 30
713 714 6
714 590 9
680 715 25
709 646 18
676 690 21
716 694 12
691 699 22
717 712 27
718 675 23
707 719 9
682 720 15
720 657 30
697 721 19
722 723 10
708 599 16
724 725 6
726 718 24
705 727 28
728 729 24
730 627 30
729 716 20
731 732 20
687 724 13
733 681 10
734 735 10
735 736 5
698 737 18
732 738 29
739 730 13
740 741 5
710 742 13
741 728 19
742 743 28
744 745 13
737 722 22
727 678 14
723 746 19
747 748 14
688 704 10
639 749 25
750 751 15
752 753 27
754 747 29
738 629 18
701 740 17
755 756 10
703 757 19
751 734 12
721 693 29
736 758 29
759 733 17
760 761 19
762 759 28
746 763 30
689 713 5
764 765 7
766 755 25
767 766 28
757 645 15
743 768 15
769 770 7
765 717 14
719 684 13
756 731 10
770 726 9
761 771 9
748 764 30
715 772 23
773 769 5
774 752 14
711 760 23
775 767 20
776 775 29
753 777 22
745 778 29
772 779 5
763 744 30
780 781 30
782 783 19
784 785 18
768 784 5
786 754 14
787 780 25
788 787 7
789 774 11
790 750 18
749 739 30
779 773 20
791 792 27
793 794 9
795 796 10
785 582 6
797 798 22
799 782 14
800 801 15
802 803 5
778 804 8
783 786 30
805 776 13
806 788 28
725 807 27
758 800 15
808 795 22
796 809 25
810 793 21
811 810 18
771 812 13
813 814 10
809 815 10
794 816 15
817 791 15
803 790 15
801 818 6
819 820 25
777 821 5
792 822 11
823 824 27
798 825 25
826 827 15
828 829 21
830 831 21
832 833 12
834 835 21
816 819 6
804 830 14
836 832 19
821 805 7
837 838 25
807 836 13
818 806 18
835 839 13
820 840 30
841 837 9
840 842 14
843 841 24
815 844 11
845 846 16
847 848 16
849 850 26
825 834 8
851 852 14
853 797 21
838 854 15
839 849 17
848 845 30
855 856 26
857 828 8
858 859 11
860 861 21
831 862 23
863 789 6
812 864 7
829 865 26
866 808 18
867 868 12
869 843 8
870 847 14
871 872 18
859 871 19
864 855 15
852 873 5
861 874 18
873 875 24
846 869 14
876 863 16
814 860 10
877 867 30
878 879 20
880 866 18
824 802 14
881 880 15
882 826 13
879 883 17
884 762 19
885 817 15
886 876 23
844 853 21
887 888 26
888 877 7
889 890 28
891 823 23
892 887 19
856 813 10
875 893 21
868 894 28
781 881 9
895 886 14
850 896 14
822 897 12
898 882 30
899 895 16
900 901 12
854 889 28
872 799 12
874 902 15
903 857 19
904 905 6
906 907 20
908 892 18
909 878 29
896 910 13
911 811 17
833 912 9
865 904 25
893 913 6
914 915 20
890 916 30
883 914 17
897 909 21
917 903 25
916 898 25
894 870 26
918 917 21
919 858 11
915 918 16
920 921 6
922 923 9
921 891 13
924 899 23
912 924 6
907 925 22
925 926 21
827 927 21
928 929 18
930 931 8
932 908 29
933 922 10
913 930 24
934 935 18
936 928 16
862 937 19
938 900 25
937 939 27
939 919 20
927 940 26
931 941 18
942 906 26
943 944 10
945 932 30
842 946 29
941 943 25
947 934 27
948 949 25
950 911 19
923 945 14
902 947 26
944 885 16
935 951 7
949 952 19
901 953 24
905 948 17
926 954 11
955 920 6
956 936 27
910 957 10
958 851 5
954 959 30
960 958 14
961 962 18
963 942 6
964 965 30
929 884 18
957 950 7
946 956 8
959 966 12
967 968 11
962 938 10
969 961 23
968 970 11
971 972 20
973 974 29
975 976 29
977 960 15
978 979 22
980 981 10
981 982 28
982 983 25
976 984 28
985 986 20
987 980 10
951 963 11
988 973 29
972 955 8
984 989 23
952 975 13
974 971 20
990 985 30
986 988 20
970 991 22
965 933 9
979 969 21
991 987 13
940 967 8
989 978 14
966 964 30
992 977 8
983 993 9
994 990 27
995 996 14
993 992 26
997 998 13
953 997 13
999 994 28
996 999 20
998 2 28
1 995 8
