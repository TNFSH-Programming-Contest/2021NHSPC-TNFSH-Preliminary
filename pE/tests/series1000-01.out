17166.0000000000000000000
